src/content/components/FeatureView/FeatureView.js

```diff
@@ -71,14 +72,19 @@ const FeatureBugList = ({hideIfEmpty, bugs, title, extraColumns = [], ...restPro
 };
 
 const EngineeringView = props => {
-  const {bugs} = props;
+  const {bugs, subMetas} = props;
   return (<React.Fragment>
     <FeatureBugList title="Untriaged bugs" hideIfEmpty={true} bugs={bugs.untriaged} extraColumns={["cf_status_nightly", "cf_status_beta"]} />
-
     <FeatureBugList title="Uplift candidates" hideIfEmpty={true} bugs={bugs.uplift} extraColumns={["cf_status_nightly", "cf_status_beta"]} />
-    <FeatureBugList title={`Required for Current Release (Firefox ${currentRelease})`} bugs={bugs.current} />
+    <h3>Required for Current Release (Firefox {currentRelease})</h3>
+    {subMetas.length ? [
+      ...Object.keys(bugs.currentBySubMeta).map(id => {
+        const meta = subMetas.find(m => String(m.id) === id);
+        return <FeatureBugList key={meta.id} subtitle={meta.summary} showHeaderIfEmpty={true} bugs={bugs.currentBySubMeta[meta.id]} />;
```

Comment:
We can think of making subtitle here a link so that its easier to navigate to sub meta bug in bugzilla. Right now one need to open one of the bugs to find sub meta bug id